devices/SparkfunLCD/SparkfunLCD.cs

```diff
@@ -0,0 +1,874 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
And can you as well rename the files to follow the pattern and the folder?
On Windows, it's not super straight forward: do a first rename of all the files to lowercase. Then do a commit
Then rename the folder by adding something. Do a commit
Then rename the folder to the final naming. Do a commit
All should then be renamed properly :-)